template/asyncapi.js

```diff
@@ -0,0 +1,574 @@
+import React from "react";
+import { File, Text } from "@asyncapi/generator-react-sdk";
+import * as _ from 'lodash';
+import { sample } from 'openapi-sampler';
+
+export default function MarkdownTemplate({ asyncapi }) {
+  return (
+    <File>
+      <Text>
```

Comment:
I wonder if we can get rid of this. For instance, we can make File extend Text, so it's not needed unless you want to indent stuff. What do you think?

Reply:
Yes, it's only a draft, so I will remove unnecessary things :) Thanks!